src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Because it's just a draft, and from a renderer work, it became more code spread than i expected...
I pushed this code and wait feedback from an user, and modify it after.

Why i such low value ? well, 4K transcoding make the i7 CPU on its kneels, so my plan was to not use CRF if necessary, but i guess that i will then loose the search ability if i make it VRF...

Anyway, just a draft with thoughts and tries.